src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -507,5 +507,26 @@ public static int countEmptyStrings(String... strings) {
      */
     public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
-    }  
+    }
+    
+    /**
+     * Removes the outermost enclosing square brackets surrounding a string.
+     * 
+     * @param str
+     * @return the string without the outermost enclosing square brackets and 
+     *         without leading and trailing spaces, if the given string is enclosed by square brackets <br/> 
+     *         the string itself if the given string is not enclosed by square brackets <br/>
+     *         null if the given string is null
+     */
```

Comment:
Question: why does this method do trimming? It's job is to remove enclosing brackets. May be we should just stick to that?. The caller can do pre and post trimming as necessary?